x-pack/legacy/plugins/infra/public/pages/logs/analysis/setup/initial_configuration_step/analysis_setup_indices_form.tsx

```diff
@@ -11,12 +11,10 @@ import React, { useCallback, useMemo } from 'react';
 
 export type IndicesSelection = Record<string, boolean>;
 
-export type IndicesValidationError = 'TOO_FEW_SELECTED_INDICES';
-
 export const AnalysisSetupIndicesForm: React.FunctionComponent<{
   indices: IndicesSelection;
   onChangeSelectedIndices: (selectedIndices: IndicesSelection) => void;
-  validationErrors?: IndicesValidationError[];
+  validationErrors?: string[];
```

Comment:
Isn't there still a well-known list of errors that we can narrow it down to here? That way we can unsure we don't pass an error which we don't know how to render.

Reply:
Take into account that now we are passing the messages here, not the error types. If we end up moving the messages out of the API response then it makes sense to type this indeed.